src/utils.ts

```diff
@@ -98,58 +109,45 @@ export function getMarkerPopupHtml(): string {
     `;
 }
 
+/**
+ * @hidden
+ * @internal
+ */
+export interface HtmlMarkerOptions {
+    big: boolean;
+    interactive: boolean;
+}
+
 /**
  * @hidden
  * @internal
  */
 export function getHtmlMarker(
     map: mapgl.Map,
     coordinates: GeoPoint,
-    big: boolean,
+    opts: HtmlMarkerOptions,
```

Comment:
Опасное название типа, легко спутать с опциями HTML-маркера. Давай просто в ф-е его определим, типа так
```suggestion
    opts: {
        big?: boolean;
        interactive?: boolean;
    },
```
ну и опции можно сделать необязательными, undefined, значит false